webcompat/templates/issue-wizard-form.html

```diff
@@ -10,380 +10,18 @@ <h1 class="headline-1">Report Site Issue</h1>
   novalidate
 >
   <h3 class="page-heading">Filing a web compatibility issue</h3>
```

Comment:
pretty much no changes here, just split this file into smaller html files

Reply:
yeah, makes it easier to digest.